mapcontrol.js

```diff
@@ -55,6 +55,12 @@ function createMapControl(elementName) {
         },
 
         addTask: function (coordinates, names) {
+            //Clearer if we don't show track to and from start line and finish line, as we are going to show lines
+            //So throw away first and last points
+            coordinates.shift();
+            names.shift();
+            coordinates.pop();
+            names.pop();
```

Comment:
I agree that we no longer want the lines from take-off to start and from finish to landing.

However, I think it would be neater to exclude the take-off and landing points from the task array in the first place, rather than throwing them away immediately before plotting. (I.e. Add some logic to parseigc.js to skip over the task lines where the coordinates are all zeroes.)

As this currently stands, the addTask function has the side-effect of altering the contents of the array passed into it, which the caller might not expect.